app/server/lib/DocApi.ts

```diff
@@ -1178,16 +1183,22 @@ export class DocWorkerApi {
       })
     );
 
-    // Create a document.  When an upload is included, it is imported as the initial
-    // state of the document.  Otherwise a fresh empty document is created.
-    // A "timezone" option can be supplied.
-    // Documents are created "unsaved".
-    // TODO: support workspaceId option for creating regular documents, at which point
-    // existing import endpoint and doc creation endpoint can share implementation
-    // with this.
-    // Returns the id of the created document.
+    /**
+     * Create a document.
+     *
+     * When an upload is included, it is imported as the initial state of the document.
+     * Otherwise, the document is left empty.
+     *
+     * If a workspace id is included, the document will be saved there instead of
```

Comment:
As a convenience, the API will generally accept the strings in place of an orgId, which is very handy. Workspaces I always find more awkward to work with, since I have to go look up their numeric ids. I wonder if we could accept strings also here as a convenience.

I guess that is a separate feature, wouldn't make sense to slow this down.

Reply:
I don't see a problem with doing it, but I agree it's probably a separate feature. I'm assuming we'll want to update all our other API endpoints to also accept a workspace name, so we can bundle all of that into a separate PR.